doc/nrf/libraries/dfu/pcd.rst

```diff
@@ -28,8 +28,8 @@ This is done to avoid compromised code from being able to perform updates of the
 The network core uses the PCD library to look for instructions on where to find the updates.
 Once an update instruction is found, this library is used to transfer the firmware update image.
 
-On the application core, the PCD library is used by the :doc:`mcuboot:index-ncs` sample.
-On the network core, the PCD library is used by the :ref:`nc_bootloader` sample.
+On the application core, the this library is used when having :ref:`MCUboot as an immutable bootloader <ug_bootloader_adding_upgradable_mcuboot>`.
```

Comment:
@hakonfam Is PCD only used by MCUboot when it is the immutable bootloader? When NSIB is the immutable one, PCD is still used by MCUboot, right?

Reply:
I understand that PCD is used by MCUboot and not NSIB. What I mean is that the doc is saying:
_[PCD] is used when having MCUboot as an immutable bootloader_

If I understand correctly, PCD is used whether MCUboot is immutable or not (i.e. whether NSIB is the immutable BL or not).

Reply:
That's correct. PCD is used in MCUBoot in either case.

Reply:
Then I propose this instead:
<code>On the application core, this library is used by :ref:\`MCUboot <ug_bootloader_adding_upgradable_mcuboot>\`.</code>